doc/man7/RSA-PSS.pod

```diff
@@ -26,82 +15,26 @@ It has associated private key and public key formats.
 This algorithm shares several control operations with the B<RSA> algorithm
 but with some restrictions described below.
```

Comment:
They're not described below any more.

Reply:
Fixing

Reply:
Not fixed?

Reply:
Yes, fixed.  I added text that describes the restrictions.  See

Reply:
Ah! Ok.